src/containers/Posts/updatePost/updatePost.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Input from '../../../components/UI/Input/Input';
+import Button from '../../../components/UI/Button/Button';
+import classes from './updatePost.css';
+import * as actions from '../../../store/actions/index';
+import Spinner from '../../../components/UI/Spinner/Spinner'
+class updatePost extends Component {
+
+    inputChangedHandler = (event) => {
+        console.log("[inputChangedHandler]", event.target)
+    }
+
+    submitHandler = (event) => {
+        event.preventDefault();
+        this.props.updatePost(event.target.title.value, event.target.content.value, this.props.match.params.id, this.props.token, this.props.userId);
+    }
+
+    render() {
+        let postDetails = this.props.postDetails
+        var title, content
+
+        postDetails.map(post => {
+            title = post.title,
+                content = post.content
+        })
+
```

Comment:
No validation on update post, please add validations.

Reply:
Fixed this issue.